packages/coins/src/evm/erc20.json

```diff
@@ -128046,6 +128046,21 @@
       "en": ""
     }
   },
+  {
+    "id": "tron_-bsc",
```

Comment:
This is supposed to be the coingecko id

Reply:
changed undo.

added version field for the new entry.